projects/iio_demo/src.mk

```diff
@@ -86,3 +86,42 @@ INCS += $(PROJECT)/src/hal/stm32/main.h \
 
 LSCRIPT = $(PROJECT)/src/linker/STM32F446RETX_FLASH.ld
 endif
+
+ifeq (linux,$(PLATFORM))
+
+SRC_DIRS += $(NO-OS)/libraries/mbedtls/include/mbedtls
+SRCS += $(NO-OS)/network/linux_socket/linux_socket.c 
```

Comment:
nitpick: maybe commit order should change here because linux_socket does not exist yet by this point.